README.md

```diff
@@ -19,6 +19,7 @@ Set up the latest or a specific version of [Keepalived](http://www.keepalived.or
 * `keepalived_git_repo`: [default: `https://github.com/acassen/keepalived.git`]: Keepalived git repo
 * `keepalived_version`: [default: `v2.2.2`]: Keepalived version to install
 
+* `keepalived_install_from`: [optional, default `git`]: how to install keepalived (actually it's support git/repo)
```

Comment:
Maybe call this `keepalived_install_method` (with values `native` and `git`)

Reply:
Fixed